src/averageAges.js

```diff
@@ -14,14 +14,23 @@
  *
  * @return {number}
  */
+const MALE = 'f';
+const FEMALE = 'f';
```

Comment:
The constant `MALE` is incorrectly set to 'f'. It should be 'm' to accurately represent male individuals in the dataset.